### PR TITLE
D sminorcorrections

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Correction on Pawli in DS Ethiop 11. Could you please change the clavis number in the "Introduction"?  It is mentioned in note 31 (M. Villa can be briefly thanked for hints concening the identity of the text).

### CODE REVIEW

File: DayrSuryan/DSEthiop19.xml [hunk 1]
```diff
@@ -289,7 +289,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                              The scribe executed the word divider <foreign xml:lang="gez">፡</foreign>
                              in a peculiar way. He interpreted as dot
                              the last letter element protruding to the right,
-                             (cp., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
+                             (cp., e.g., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
```
Comment (thea-m) [inline DayrSuryan/DSEthiop19.xml:292]: Is this Oxford style? I find them a lot of abbreviations and commas..I was told that no comme should be used after e.g.

Comment (thea-m) [reply DayrSuryan/DSEthiop19.xml:292]: @SusanneHummel can you help us one last time in this regard :) then we will hopefully learn

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
Sorry, @DenisNosnitsin1970 , but I can see no new Clavis number assigned here?

File: DayrSuryan/DSEthiop19.xml (hunk 1 above)
Comment (SusanneHummel) [reply DayrSuryan/DSEthiop19.xml:292]: there should be no comma after e.g.
I don't have the book here. i think my copy is still with @DenisNosnitsin1970 
I once made a photo of this entry from the manual for a BodCat Ms record. Unfortunately, I don't remember which one...
Why do you need cp?

Comment (thea-m) [reply DayrSuryan/DSEthiop19.xml:292]: Thank you!

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop19.xml:292]: I see in the Oxford Style book that no comma follows e.g. But now I see that "e.g." is not necessary here. I am deleting it.

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop19.xml]: Cp. for "compare"?

**Comment by DenisNosnitsin1970** [comment]
New number is in Ethiop. 11, went yesterday.

File: DayrSuryan/DSEthiop19.xml (hunk 1 above)
Comment (SusanneHummel) [reply DayrSuryan/DSEthiop19.xml]: I found both cp. and e.g. superfluous. As you also noticed.

**Review by thea-m — APPROVED** [review]

**Pull request merged**